source/info/bockowsk/Book.java

```diff
@@ -0,0 +1,16 @@
+package info.bockowsk;
+import java.io.*;
+
+abstract class Book implements Serializable {
+    final String autor;
+    final String tytul;
+    final String wydawca;
+    String tresc;
+
+    Book(String a, String t, String w) {
```

Comment:
You should give variables appropriate names, that do not introduce confusion. I.e. `author` instead of `a` etc.

When you look at such constructor `Book(String a, String t, String w)` you have absolutely no idea what the class is asking you. Its a "WTF" and you don't like them:

![image](https://user-images.githubusercontent.com/3918865/33826422-41c528c6-de65-11e7-9228-f29062826b85.png)

Reply:
Thank you for the feedback, I was always writing solely for myself and I've never thought it is very obscure. I am going to change this wrong habit and name variables in a meaningful way.